ironfish/src/wallet/account.ts

```diff
@@ -193,7 +191,7 @@ export class Account {
           blockHash,
           sequence,
           submittedSequence,
-          timestamp,
+          timestamp: blockHeader.timestamp,
```

Comment:
This means that transactions will move in your `wallet:transactions` list, is that ok? Do we intend that behavior?